### PR TITLE
Added empty check for magmi_mixin.php file on array

### PR DESCRIPTION
**Pull request by ChameleonDevil**
Please read below comment for details.

### CODE REVIEW

File: magmi/inc/magmi_mixin.php [hunk 1]
```diff
@@ -25,7 +25,7 @@ public function __call($data, $arg)
         if (substr($data, 0, 8) == "_caller_") {
             $data = substr($data, 8);
         }
-        $ccallers = count($this->_callers);
+        $ccallers = empty($this->_callers) ? 0 : count($this->_callers);
```
Comment (tmotyl) [inline magmi/inc/magmi_mixin.php]: shoudlnt we just initialize the variable like `protected $_callers = array();`
Can you please check if this approach also work for you? sounds more elegant to me.

Comment (tmotyl) [reply magmi/inc/magmi_mixin.php]: see similar patch: https://github.com/dweeves/magmi-git/pull/587/files

**Comment by ChameleonDevil** [comment]
Hi, thank you.
Sorry tmotyl, I still wanted to reply after applying the update,  I see it is already merged.
I think it should work.

**Comment by tmotyl** [comment]
Perfect, thanks for letting me know

**Pull request merged**